jobs/debug-pod.Jenkinsfile

```diff
@@ -0,0 +1,187 @@
+import org.yaml.snakeyaml.Yaml;
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+    libcloud = load("libcloud.groovy")
+}
+
+// Base URL through which to download artifacts
+BUILDS_BASE_HTTP_URL = "https://builds.coreos.fedoraproject.org/prod/streams"
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             choices: pipeutils.get_streams_choices(pipecfg),
+             description: 'CoreOS stream to build'),
+      choice(name: 'ARCH',
+             description: 'The target architecture',
+             choices: pipeutils.get_supported_additional_arches()),
+      string(name: 'COREOS_ASSEMBLER_IMAGE',
+             description: 'Override coreos-assembler image to use',
+             defaultValue: "",
+             trim: true),
+      string(name: 'TIMEOUT',
+             description: 'Timeout value',
+             defaultValue: "1h",
+             trim: true),             
+
+    ]),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+def build_description = "[${params.STREAM}][${params.ARCH}]"
+
+// Reload pipecfg if a hotfix build was provided. The reason we do this here
+// instead of loading the right one upfront is so that we don't modify the
+// parameter definitions above and their default values.
+if (params.PIPECFG_HOTFIX_REPO || params.PIPECFG_HOTFIX_REF) {
+    node {
+        pipecfg = pipeutils.load_pipecfg(params.PIPECFG_HOTFIX_REPO, params.PIPECFG_HOTFIX_REF)
+        build_description = "[${params.STREAM}-${pipecfg.hotfix.name}][${params.ARCH}]"
+    }
+}
+
+// runtime parameter always wins
+def cosa_img = params.COREOS_ASSEMBLER_IMAGE
+cosa_img = cosa_img ?: pipeutils.get_cosa_img(pipecfg, params.STREAM)
+
+def stream_info = pipecfg.streams[params.STREAM]
+
+def cosa_controller_img = stream_info.cosa_controller_img_hack ?: cosa_img
+cosa_controller_img = 'quay.io/dustymabe/coreos-assembler-staging:cosa-tmux'
+// DELETE ^^
```

Comment:
would be nice if we could go ahead and get the pieces in COSA that we need and stop using this image I built.

Reply:
Just to be clear, we have added tmux in [coreos-assembler](https://github.com/coreos/coreos-assembler/blob/main/src/build-deps.txt), in that case does it need to be pointed to the latest image or do we delete this?
`cosa_controller_img = http://quay.io/coreos-assembler/coreos-assembler:latest`

Reply:
oh good. Forgot we did that in https://github.com/coreos/coreos-assembler/pull/3404

then just delete line 55 and 56.

honestly lets just delete line 54 too and change line 97 to use `cosa_img`.

Reply:
done